src/classes/astyle.cpp

```diff
@@ -1,13 +1,24 @@
 #include "astyle.h"
 #include <QStyle>
+#include <QtGlobal>
 #include <QStyleFactory>
 #include <QApplication>
 #include "src/testing/adebug.h"
 #include "src/classes/asettings.h"
 
 // this was throwing hundreds of warnings on 5.9.5, fusion is a good default for all platforms, let's
 // stick with that for now.
+// [G]: Ensure QStringLiteral is in version
+#if QT_VERSION > QT_VERSION_CHECK(5, 9, 0)  // we can set macros like STABLE_QT or smth for readability of course
```

Comment:
While being a good solution from a code-perspective it is not addressing the second point of my comment:

`fusion is a good default for all platforms, let's stick with that for now`
I take responsibility since I put the other reason first - But Fusion is the default out-of-the-box theme for qt on all platforms. It is universally supported by all versions of qt5 on all platforms, so it should be the default choice.

Same note on the logic as above